src/mnode/mnode-pastel.h

```diff
@@ -351,7 +351,7 @@ class CArtSellTicket : public CPastelTicket
     std::string ToJSON() const noexcept override;
     std::string ToStr() const noexcept override;
     bool IsValid(bool preReg, int depth) const override;
-    CAmount TicketPrice(const unsigned int nHeight) const noexcept override { return askedPrice/50; }
+    CAmount TicketPrice(const unsigned int nHeight) const noexcept override { return std::max(10u, askedPrice / 50); }
```

Comment:
I am not sure what is the correct calculation.
But what is the expected TicketPrice if askedPrice is, for example: 2545? I think it should be 51, instead of 50, right?